spec/models/video_spec.rb

```diff
@@ -35,4 +36,25 @@
       expect(Video.search_by_title "").to eq([])
     end
   end
+
+  describe "#calculate_rating" do
+    let(:video) { Fabricate(:video) }
+
+    it "returns 0 if there are no reviews" do
```

Comment:
Be careful of out of sync expectations vs spec descriptions. This description is technically not accurate.